src/apiary.apib

```diff
@@ -892,6 +896,23 @@ View all Campaigns that the Publisher is associated with, along with the status
     + Attributes
         + campaigns (array[Publisher Campaign Wrapper])
 
+## Publisher Campaign Tracking [/user/publisher/{publisher_id}/campaign/{status}/tracking]
+
++ Parameters
+    + publisher_id: 1l1007802 - ID of the publisher
+    + status: (string) - The status of the publisher on the campaign
+        + a - approved
```

Comment:
If `n` isnt valid, then we you could create a new status for a/p/r and `Include` that into this existing Status (enum)